Formula/qt5.rb

```diff
@@ -27,28 +26,6 @@ class Qt5 < Formula
     sha256 "73d33dd2563c39542844c276a7bd43463f2974fde141e7afeb3057168adbe606" => :mavericks
   end
 
-  # Restore `.pc` files for framework-based build of Qt 5 on OS X. This
```

Comment:
poppler qt5 option fixed in https://github.com/Homebrew/homebrew-core/pull/5864
mkvtoolnix qt5 option fixed in https://github.com/Homebrew/homebrew-core/pull/5865